src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -672,6 +679,62 @@ public override Conversion GetMethodGroupConversion(BoundMethodGroup source, Typ
             return conversion;
         }
 
+        protected override bool HasImplicitTupleConversion(BoundExpression source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            if (source.Kind != BoundKind.NaturalTupleExpression)
+            {
+                // source must be a natural tuple expression
+                return false;
+            }
+
+            // strip nullable from the destination
+            // the following should work
+            //    (int, double)? x = (1,2);
+            if (destination.IsNullableType())
```

Comment:
Regular  literals  handle this by making it "ImplicitNullable" conversion and leaving underlying literal as-is. 
I am currently classifying the whole thing as TupleConversion and handle nullable when conversion node is created.

I am considering changing to the same approach as with regular literals (i.e. classify this as ImplicitNullable conversion and special-case later), but it is not as straightforward since tuples are not constant expressions and tuple types do not convert other than via identity conversions.